app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java

```diff
@@ -102,11 +102,10 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                         swActiveSecurity.setChecked(true);
                         SP.putBoolean(getString(R.string.preference_use_password), true);
                         toggleEnabledChild(true);
-                    }
-                    else
-                    setPasswordDialog();
-                }
-                else {
+                        Snackbar.make(findViewById(android.R.id.content), "Password Set", Snackbar.LENGTH_SHORT)
+                                .show();
+                    } else setPasswordDialog();
+                } else {
```

Comment:
Cant you use SanckBar Handler here as it is used throughout the code. Also put "password set" string into strings.xml instead of hardcoding

Reply:
I will make use of snackbar handler when #2627 is merged. I will fix this along with issue #2593.